lib/ansible/playbook/conditional.py

```diff
@@ -107,83 +102,22 @@ def evaluate_conditional_with_result(self, templar: Templar, all_vars: dict[str,
 
         return result, false_condition
 
-    def _check_conditional(self, conditional, templar, all_vars):
-        '''
-        This method does the low-level evaluation of each conditional
-        set on this object, using jinja2 to wrap the conditionals for
-        evaluation.
-        '''
-
+    def _check_conditional(self, conditional: str, templar: Templar, all_vars: dict[str, t.Any]) -> bool:
         original = conditional
 
         if templar.is_template(conditional):
             display.warning('conditional statements should not include jinja2 '
                             'templating delimiters such as {{ }} or {%% %%}. '
                             'Found: %s' % conditional)
 
-        # make sure the templar is using the variables specified with this method
         templar.available_variables = all_vars
-
         try:
-            # if the conditional is "unsafe", disable lookups
-            disable_lookups = hasattr(conditional, '__UNSAFE__')
-            conditional = templar.template(conditional, disable_lookups=disable_lookups)
-
-            if not isinstance(conditional, text_type) or conditional == "":
+            conditional = templar.template(conditional)
```

Comment:
no need to template the conditional here, we are doing it below after interpolation, this changes teh result as then we x2 template the conditonal and it's result